input/fsh/EX_Example1_Genomics.fsh

```diff
@@ -5,7 +5,7 @@ Description: "mCODE Example for Cancer Genetic Variant"
 * meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/mcode-cancer-genetic-variant"
 * status = #final "Final"
 * method = LNC#LA26398-0 "Sequencing"
-// valueCodeableConcept has alternate codings depending on where to place the interpretation of "Positive or Negative".
+// value[x] has alternate codings depending on where to place the interpretation of "Positive or Negative".
 // * valueCodeableConcept = SCT#10828004 "Positive (qualifier value)"
```

Comment:
I think whenever we have something commented out in FSH, we should have a comment explaining why it's commented out.